test/integration/test-common.c

```diff
@@ -582,3 +580,14 @@ test_esys_teardown(TSS2_TEST_ESYS_CONTEXT *test_ctx)
         free(test_ctx);
     }
 }
+
+void
+test_fapi_esys_teardown(TSS2_TEST_ESYS_CONTEXT *test_ctx)
```

Comment:
I'm a little confused by this. We already have  `test_esys_teardown()` and `test_fapi_teardown()`. This new function `test_fapi_esys_teardown()` seems to be never called. Maybe you added this accidentally?